microservices/auth/src/server.ts

```diff
@@ -22,10 +23,19 @@ if (
 }
 
 const PORT = process.env.PORT !== undefined ? Number(process.env.PORT) : 8080;
-
+const AUTH_MONGODB_URI = fs.existsSync(process.env.AUTH_MONGODB_URI)
+  ? fs.readFileSync(process.env.AUTH_MONGODB_URI, "utf8").trim()
+  : process.env.AUTH_MONGODB_URI;
+const AUTH_SESSION_SECRET = fs.existsSync(process.env.AUTH_SESSION_SECRET)
+  ? fs.readFileSync(process.env.AUTH_SESSION_SECRET, "utf8").trim()
+  : process.env.AUTH_SESSION_SECRET;
 const useLocalhost = process.env.USE_LOCALHOST === "1";
-const USER_SERVICE_URL = useLocalhost ? "http://localhost:3219" : "";
-const QUESTION_SERVICE_URL = useLocalhost ? "http://localhost:3001" : "";
+const USER_SERVICE_URL = process.env.USER_SERVICE_URL
```

Comment:
You can consider using the [Nullish coalescing operator `??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) instead. (`process.env.RANDOM_VALUE` gives `undefined` if the value is not found in the `.env` file)

```js
const USER_SERVICE_URL = process.env.USER_SERVICE_URL ?? "http://localhost:3219"
````